src/EPR.RegulatorService.Facade.Core/Helpers/UrlHelpers.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace EPR.RegulatorService.Facade.Core.Helpers
+{
+    public static class UrlHelpers
+    {
+        private static readonly string[] _allowedSchemes = { "https", "http" };
+        private static readonly string[] _allowedDomains = { "http://localhost" };
```

Comment:
This is not correct, this check is meaningless as there is not Schema or Domain present in the constructed Url\string, what SQ is suggesting as a fix is that we need to check that the user inputs are not altering the domain name and schema, and this can't happen as the base url is already provided to _httpClient and it's not part of this string

personally I sugget you disable this as false positive as suggested here

https://community.sonarsource.com/t/change-this-code-to-not-construct-the-url-from-user-controlled-data/34393/6